src/pocketmine/event/block/SignChangeEvent.php

```diff
@@ -73,6 +73,13 @@ public function getLine($index){
 	}
 
 	/**
+	 * @param string[] $lines
+	 */
+	public function setLines(array $lines){
+		$this->lines = $lines;
```

Comment:
If assertions are turned off, the user doesn't mind about faulty code, so no need to check. This is not a critical check that must be executed all the time.